src/frontends/onnx/frontend/src/op/com.microsoft/ONNX_Frontend_extension.cpp

```diff
@@ -0,0 +1,110 @@
+#include "ONNX_Frontend_extension.hpp"
+#include <iostream>
+
+namespace onnx_opset {
+    namespace op {
+        void Pad(const float* input_data, const int64_t* pads_data, float* output_data, const std::vector<int64_t>& input_shape, const std::vector<int64_t>& output_shape, const std::string& mode, float constant_value) {
```

Comment:
Thank you for contibution! Be aware that translation ONNX op to OpenVINO representation has be to be a function in shape:
```
OutputVector pad(const Node& node) {
return OneOrMoreOutoutOfOvSubgraph
}
```
In particular we can't use input tensors data (like `input_data` in your case) if not constants.
Please look at https://github.com/openvinotoolkit/openvino/blob/50897e86e67b623b96e6c9a815a1606f88620b8f/src/frontends/onnx/frontend/src/op/pad.cpp#L68

Reply:
> Thank you for contribution! Be aware that translation ONNX op to OpenVINO representation has be to be a function in shape:
> 
> ```
> OutputVector pad(const Node& node) {
> return OneOrMoreOutoutOfOvSubgraph
> }
> ```
> 
> In particular we can't use input tensors data (like `input_data` in your case) if not constants. Please look at
> 
> https://github.com/openvinotoolkit/openvino/blob/50897e86e67b623b96e6c9a815a1606f88620b8f/src/frontends/onnx/frontend/src/op/pad.cpp#L68

Thank you for your review @mbencer , I will definitely correct it and let you know . Secondly I would like to know if the implementation logic ,return values and such are 
correct ? If not could you  please point me what went wrong . Thanks!